python/chatting_sample.py

```diff
@@ -0,0 +1,390 @@
+import json
+import math
+import sys
+import time
+import requests
+import wave
+import threading
+
+try:
+    import azure.cognitiveservices.speech as speechsdk
+except ImportError:
+    print("""
+    Importing the Speech SDK for Python failed.
+    Please run "pip install azure-cognitiveservices-speech" to install the SDK.
+    """)
+    sys.exit(1)
+
+
+# Set up the subscription info for the Speech Service:
+# Replace with your own subscription key and service region (e.g., "westus").
+speech_key, service_region = "YourSubscriptionKey", "YourServiceRegion"
+your_resource_name = "YourResourceName"
```

Comment:
Add `Set up the parameters for Azure OpenAI Services` before this line.

Reply:
> Add `Set up the parameters for Azure OpenAI Services` before this line.

ok, added